metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/database/SraObjectsImporterThroughDatabase.java

```diff
@@ -76,7 +137,20 @@ public Analysis importAnalysis(String accession) {
 
     @Override
     public ReferenceSequence importReferenceSequence(String accession) {
-        return super.importReferenceSequence(accession);
+        String[] accnRefNameSplit = accession.split("#", 2);
+        String refAccession = accnRefNameSplit[0];
+        String refName = accnRefNameSplit[1];
+        ArrayList<String> accessionList = new ArrayList<>(Arrays.asList(refAccession));
+        String[] refNameSplit = refName.split("\\.", 2);
+        String patch = null;
+        if (refNameSplit.length == 2) {
+            patch = refNameSplit[1];
+        }
+        ReferenceSequence referenceSequence = new ReferenceSequence(refName, patch, accessionList,  ReferenceSequence.Type.ASSEMBLY);
```

Comment:
```suggestion
        ReferenceSequence referenceSequence = new ReferenceSequence(refName, patch, accessionList, ReferenceSequence.Type.ASSEMBLY);
```

Reply:
Fixed in https://github.com/EBIvariation/metadata-ws/pull/50/commits/f992ccc9192cdef417bcfafd9a2ea1b554e32493.